src/__mocks__/react-onclickoutside.js

```diff
@@ -8,13 +8,14 @@ import * as React from 'react'
 const onClickOutside = <T> (WrappedComponent: React.ComponentType<T>) => class extends React.Component<T> {
   static displayName = `OnClickOutside(${WrappedComponent.displayName || WrappedComponent.name || typeof WrappedComponent})`
 
-  render () {
-    const component = <WrappedComponent {...this.props} />
-    // $FlowFixMe
-    if (typeof component.type.prototype.handleClickOutside !== 'function') {
+  checkRef = (instance: mixed) => {
```

Comment:
Should be `React.ComponentType<T>` right?

Reply:
Hmm no... As this is an actual instance which might or might not have the necessary attribute `handleClickOutside`, actually I think in this case `mixed` might be the best fit here; at least `React.ComponentType<T>` throws an error that it doesn't know the attribute handleClickOutside.
I tried some other types which didn't work as well...

Reply:
Hmm no... As this is an actual instance which might or might not have the necessary attribute `handleClickOutside`, actually I think in this case `mixed` might be the best fit here; at least `React.ComponentType<T>` throws an error that it doesn't know the attribute handleClickOutside.
I tried some other types which didn't work as well...